spmd/tensor/api.py

```diff
@@ -30,41 +86,53 @@ class Tensor(torch.Tensor):  # pyre-ignore[13]: pyre is bad at __new__
     @staticmethod
     def __new__(
         cls,
+        local_tensor: torch.Tensor,
```

Comment:
What if users want to pass in a global tenor and convert it to a sharded DT. So the only way is to use distribute_tensor?

Reply:
yeah that's what the `distributed_tensor` API helps, because it needs additional logic on chunking it to shards and sending it over to other ranks.

Reply:
I see. It would be nice if we can mention this in the comment or at least add a TODO for it.